lib/ret/storage_used.ex

```diff
@@ -0,0 +1,17 @@
+defmodule Ret.StorageUsed do
+  use Cachex.Warmer
+  use Retry
+
+  def interval, do: :timer.minutes(5)
+
+  def execute(_state) do
+    storage_path = Application.get_env(:ret, Ret.Storage)[:storage_path]
+    
+    case System.cmd("df", ["--output=used", storage_path]) do
+      { lines, 0 } ->
+        used = lines |> String.split("\n") |> Enum.at(1) |> Integer.parse |> elem(0)
```

Comment:
Is it safe to assume only one mounted filesystem and that the one we care about will always be the first one?

Reply:
passing the path to `ld` here should return the mount the path exists on. the main weirdness is we are going to cap storage to that entire volume -- so if its a volume shared with other data, it'll quota out early.

Reply:
updated to use `ld -k -T`